Makefile

```diff
@@ -1,9 +1,11 @@
 default: serve
 
 serve-venv:
-	@python3 -m venv venv && source venv/bin/activate \
-	&& pip install -r requirements.txt \
-	&& mkdocs serve
+ifeq ($(OS),Windows_NT)
+	@python -m venv venv && venv\Scripts\activate && pip install -r requirements.txt && mkdocs serve
+else
```

Comment:
This will probably break in MinGW.

Reply:
If someone uses mingw for their wiki devving, feel free to try it and PR a fix for it. It's really just meant as a convenience for devs. The full documentation for how to build & test is in the readme still